components/build-service/development/sealights-token-patch.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: build-service-controller-manager
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        env:
+          - name: SEALIGHTS_TOKEN
+            value: ""
```

Comment:
since we push the argocd changes during bootstrap to a public repo, wouldn't the token be exposed?

should we consider loading the token from the secret instead?

Reply:
do we want to make users to have a token locally?